prov/usnic/src/usnic_direct/vnic_rq.h

```diff
@@ -119,7 +119,7 @@ struct vnic_rq {
 #endif
 
 #ifdef CONFIG_NET_RX_BUSY_POLL
-	atomic_t bpoll_state;
+	atomic32_t bpoll_state;
```

Comment:
This shouldn't be changed to `atomic32_t`. This file is used in the kernel context and only gets compiled if in the kernel (hence the `CONFIG_NET_RX_BUSY_POLL`). This `atomic_t` actually refers to the Linux kernel `atomic_t` definition. 

The original typedef should have probably been called `ofi_atomic_t` to avoid this, but this code never gets compiled in userspace so we never noticed.